test/src/test/java/jenkins/security/ApiTokenPropertyTest.java

```diff
@@ -15,27 +15,27 @@
 import com.gargoylesoftware.htmlunit.html.HtmlPage;
 import com.gargoylesoftware.htmlunit.util.NameValuePair;
 import com.gargoylesoftware.htmlunit.util.UrlUtils;
+import hidden.jth.org.apache.http.Header;
+import hidden.jth.org.apache.http.HttpEntity;
+import hidden.jth.org.apache.http.HttpHost;
+import hidden.jth.org.apache.http.auth.AuthScheme;
+import hidden.jth.org.apache.http.auth.AuthScope;
+import hidden.jth.org.apache.http.auth.Credentials;
+import hidden.jth.org.apache.http.auth.UsernamePasswordCredentials;
+import hidden.jth.org.apache.http.client.AuthCache;
+import hidden.jth.org.apache.http.client.CredentialsProvider;
+import hidden.jth.org.apache.http.client.methods.CloseableHttpResponse;
+import hidden.jth.org.apache.http.client.methods.HttpGet;
+import hidden.jth.org.apache.http.client.protocol.HttpClientContext;
+import hidden.jth.org.apache.http.impl.auth.BasicScheme;
+import hidden.jth.org.apache.http.impl.client.BasicAuthCache;
+import hidden.jth.org.apache.http.impl.client.BasicCredentialsProvider;
+import hidden.jth.org.apache.http.impl.client.CloseableHttpClient;
+import hidden.jth.org.apache.http.impl.client.HttpClientBuilder;
```

Comment:
Probably it was not necessary for @oleg-nenashev to use `JenkinsRule.WebClient` here. I am not aware offhand of other cases where these kinds of classes are used.